frontend/src/container/NotificationWall.jsx

```diff
@@ -37,11 +41,26 @@ import GroupedNotificationItem from './GroupedNotificationItem.jsx'
 
 export const NotificationWall = props => {
   const [notificationList, setNotificationList] = useState([])
+  // INFO - GB -2022-06-05 - The no set bellow is not used because folderPath is a dictionary and the manipulations are done directly
+  const [folderPath, setFolderPath] = useState({}) // eslint-disable-line no-unused-vars
```

Comment:
Just wondering why manipulations are directly done and not using `setFolderPath`?

Reply:
For a dictionary we add and remove the keys directly at object pointer, so there is no real change in the state point of vue. Similar to Array.push() that does not change the array and when you use it your array is a const.